src/components/MessageActions/__tests__/MessageActionsBox.test.js

```diff
@@ -131,4 +133,12 @@ describe('MessageActionsBox', () => {
     const results = await axe(container);
     expect(results).toHaveNoViolations();
   });
+
+  it('should render CustomMessageActionsList if provided', async () => {
```

Comment:
How is possible that this test is passing if the component is not forwarded to the ComponentContext inside `Channel` component?

Reply:
Through the use of the `ComponentProvider` instead, this test setup does not use Channel component.